src/oop_ext/interface/_tests/test_interface.py

```diff
@@ -514,12 +516,16 @@ def m1(self):
 
 
 def testNull():
-    AssertImplements(Null(), _InterfM2)  # Not raises BadImplementationError
+    AssertImplements(
+        Null(), _InterfM2, requires_declaration=False
```

Comment:
I didn't get this test. How can `Null()` be an implementation of `_InterfM2`, which has a `m2` method?

Reply:
`Null` is supposed to mean "anything", so it implements "any" interface.

Think of `Null` as a `MagicMock`.